core/src/actions/swagger.ts

```diff
@@ -82,13 +82,7 @@ export class Swagger extends Action {
         if (!action) return;
 
         const tag = action.name.split(":")[0];
-        const formattedPath = route.path
-          .replace("/v:apiVersion", "")
```

Comment:
A bit too eager on the deletion there, I believe`.replace("/v:apiVersion", "")` should still be present 😛

Reply:
oops! updated now